code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -543,6 +543,34 @@
 	else
 		return FALSE
 
+	//Old Prototype
+/obj/item/clothing/head/helmet/space/hardsuit/ancient
+	name = "prototype hardsuit helmet"
+	desc = "Early prototype hardsuit helmet, designed to quickly shift over a user's head. Design constraints of the helmet mean it has no inbuitl cameras, thus it restricts the users visability."
+	icon_state = "hardsuit0-ancient"
+	item_state = "anc_helm"
+	armor = list(melee = 40, bullet = 5, laser = 5, energy = 0, bomb = 50, bio = 100, rad = 100, fire = 100, acid = 75)
```

Comment:
Quote these associations, I know it's everywhere else for armor lists but we're trying to get rid of that